gsitemap.php

```diff
@@ -623,7 +639,7 @@ protected function _getSupplierLink(&$link_sitemap, $lang, &$index, &$i, $id_sup
             if ($file_headers[0] != 'HTTP/1.1 404 Not Found' || $file_headers === true) {
                 $supplier_image = array(
                     'title_img' => htmlspecialchars(strip_tags($supplier->name)),
-                    'link' => 'http'.(Configuration::get('PS_SSL_ENABLED') ? 's' : '').'://'.Tools::getMediaServer(_THEME_SUP_DIR_)._THEME_SUP_DIR_.((!file_exists(_THEME_SUP_DIR_.'/'.(int)$supplier->id.'-'.ImageType::getFormattedName('medium').'.jpg')) ? $lang['iso_code'].'-default' : (int)$supplier->id).'-'.ImageType::getFormattedName('medium').'.jpg'
+                    'link' => $this->context->link->getBaseLink().Tools::getMediaServer(_THEME_SUP_DIR_)._THEME_SUP_DIR_.((!file_exists(_THEME_SUP_DIR_.'/'.(int)$supplier->id.'-'.ImageType::getFormattedName('medium'))) ? $lang['iso_code'].'-default' : (int)$supplier->id).'-'.ImageType::getFormattedName('medium')
```

Comment:
Isn't the link missing `.jpg` now?

Reply:
The file extension is already added, no need to hard code it :-)

Reply:
Sorry, you are right. There are some issues with suppiers. I will look into that.

I try will see if I can solve it using this: <code>$image_link = $this->context->link->getSupplierLink($supplier, $supplier->link_rewrite, (int)$supplier->id_image, ImageType::getFormattedName('medium'));</code>

Reply:
This makes no sense, I think we should remove manufacturer and supply from sitemap. They both have no pages. Has this changed from 1.6 to 1.7?

Reply:
this should be the link for instance: <code>https://localhost/1741/2_graphic-corner</code>

But it though a 404 error.

Reply:
Please check this commit. I removed supplier and manufacturer because they are not used as pages I think?

Reply:
https://github.com/PrestaShop/gsitemap/pull/53/commits/91de6e87abad28c80692137bab532cf1e392ee13